### ISSUE TITLE
[UI] Taxonomy terms index, create, edit and delete pages

### ISSUE THREAD

**Issue by NuwanJ** [cepdnaclk/portal.ce.pdn.ac.lk#126]
> Start after #125 

### UI, Routes and LiveWire Components
 
- `/taxonomy/:taxonomy/terms/`: List taxonomy terms under parent taxonomy
 - Create a route file with breadcrumb
 - Livewire row
   - Columns: code, name, created_by, updated_by 
   - Buttons: 
     - Edit
     - Delete
  - UI Similar to a standard index page 
  - No need for filters
  - No pagination 
  - No hierarchical organization (for now, this will be covered in a separate ticket)

### UI Only

- `/taxonomy/:taxonomy/terms/create` 
- `/taxonomy/:taxonomy/terms/edit/:taxonomy_term`

  - It is better to create components for each property data type for easy reuse
  - Metadata fields must be rendered based on the taxonomy's properties 
  - Edit and Delete pages should like the following: 

![Image](https://github.com/user-attachments/assets/ba582fd4-821f-4185-9a5d-135caf112429)
